articles/ctod.dd

```diff
@@ -637,6 +637,21 @@ void dostring(string s)
        to generate a fast lookup scheme for it, eliminating the bugs
        and time required in hand-coding one.
 
+       Another possibility of associating a string to a number is to
+       use an associative array.
+
+----------------------------
```

Comment:
make this a runnable example?

Reply:
Ok, I will look how to do a runnable example.